corehq/form_processor/models.py

```diff
@@ -174,7 +177,7 @@ def is_submission_error_log(self):
         return self.state == self.SUBMISSION_ERROR_LOG
 
     @property
-    @memoized
+    @cache_return_value
```

Comment:
sounds good, will let you respond to danny, otherwise i'm signed off. i see now how this a replacement for that `cached_attachments`